static/js/panels.js

```diff
@@ -116,6 +116,14 @@ exports.initialize = function () {
         }
         resize_app();
     });
+
+    // Configure keyup handlers.
+    $(".alert-link[role=button]").on("keyup", function (e) {
```

Comment:
This should ideally have been done with the same style as the last click handler.  This can be important if we end up rendering new panels after `initialize` is called.

Reply:
Yeah, you're right.